lib/user_space/user_x86_64_atomic.h

```diff
@@ -31,15 +31,20 @@
 /**
    @addtogroup atomic
 
-   Implementation of atomic operations for Linux user space uses x86_64 assembly
-   language instructions (with gcc syntax). "Lock" prefix is used
-   everywhere---no optimisation for non-SMP configurations in present.
+   Implementation of atomic operations for Linux user space uses x86_64/aarch64 
```

Comment:
Please drop aarch64-related stuff from this file - it's not needed anymore.

Reply:
Removed